CMakeLists.txt

```diff
@@ -88,6 +89,9 @@ add_subdirectory(src)
 install(FILES ${PROJECT_BINARY_DIR}/include/RAJA/raja-config.cmake DESTINATION share/cmake/raja)
 
 if(RAJA_ENABLE_TESTS)
-  enable_testing()
   add_subdirectory(test)
 endif()
+
+if(RAJA_ENABLE_EXAMPLES)
```

Comment:
This is meant to support basic examples we will build to illustrate RAJA concepts, correct?